extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -99,10 +96,11 @@ VertxWebRouterBuildItem initializeRouter(VertxHttpRecorder recorder,
     ServiceStartBuildItem finalizeRouter(
             VertxHttpRecorder recorder, BeanContainerBuildItem beanContainer,
             Optional<RequireVirtualHttpBuildItem> requireVirtual,
-            InternalWebVertxBuildItem vertx,
-            LaunchModeBuildItem launchMode, ShutdownContextBuildItem shutdown, List<DefaultRouteBuildItem> defaultRoutes,
-            List<FilterBuildItem> filters, VertxWebRouterBuildItem router, EventLoopCountBuildItem eventLoopCountBuildItem,
-            HttpBuildTimeConfig httpBuildTimeConfig,
+            InternalWebVertxBuildItem vertx, LaunchModeBuildItem launchMode, ShutdownContextBuildItem shutdown,
+            List<DefaultRouteBuildItem> defaultRoutes, List<FilterBuildItem> filters,
+            VertxWebRouterBuildItem router, EventLoopCountBuildItem eventLoopCountBuildItem,
+            HttpBuildTimeConfig httpBuildTimeConfig, HttpConfiguration httpConfiguration,
+            BuildProducer<ReflectiveClassBuildItem> reflectiveClassBuildItemBuildProducer,
```

Comment:
Let's be consistent with the names of the parameters. None of the other parameters contains `BuildItem` in them, nor is `Producer` used for producers in other methods of the class.

Sorry for these nitpicks, but I strongly think we should try and be as consistent as possible, especially when it doesn't come at the cost of anything else.